### DISCUSSION TITLE
boot into safe mode

### DISCUSSION THREAD

**Discussion by leyleyla** [ventoy/Ventoy#2165] [Ideas]
feature request: when localbooting from ventoy into windows, there should be an option to boot directly into safe mode (if possible).
safe mode is useful when standard desktop is not working due to an error

**Comment by uuaws2** [comment]
That would be a nice feature but they cant do that because  windows or linux (Some of the operation systems) OS's can only get into the safe mode when the OS is downloaded to the disk . (Im just talking from my experiance with the computers)

**Reply by uuaws2** [reply]
After doing some reaserch i found out that ventoy tool already has secure booting supported. You just need to search how to do it now.
